src/hyrdate.js

```diff
@@ -1,20 +1,28 @@
 class Hydration {
-  constructor(hydrationData) {
-    this.userId = hydrationData.userID,
-    this.date = hydrationData.date,
-    this.numOunces = hydrationData.numOunces
+  constructor(hydrationData, userID) {
+    this.userID = userID;
+    this.hydrationData = hydrationData
   }
 
-  calculateAverageDailyConsumption() {
-
+  calculateAverageTotalConsumption() {
```

Comment:
This method is calculating the average daily consumption of all users, it has to calculate the overall average for only one particular user identified by their userId.  So it will need a parameter of the userID and the function will have to be modified to only include the numbers of that user.  The test will also need to be adjusted to reflect this.